course_inventory/inventory.py

```diff
@@ -182,6 +182,31 @@ def gather_course_data_from_api(account_id: int, term_ids: Sequence[int]) -> pd.
     return course_df
 
 
+def gather_account_data_from_api(account_ids: Sequence[int]) -> pd.DataFrame:
+    logger.info('** gather_account_data_from_api')
+    url_ending_with_scope = f'{API_SCOPE_PREFIX}/accounts/'
```

Comment:
No other methods are using this endpoint. Are you saying that you want me to put it in all caps?

Reply:
Yeah, I would.

Reply:
Even though this is only used in the local function scope? I tend to use ALL_CAPS for global variables and basically nothing else.

Reply:
I'm going to defer making this change, since it would probably require changes elsewhere in this file for consistency. Also, I think I'd like to have this discussion with you and other developers, since I've never done variable naming this way outside of the module level. Relevant link: https://www.python.org/dev/peps/pep-0008/#constants